composer.json

```diff
@@ -6,21 +6,21 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7",
-        "php": ">=7.2",
-        "nesbot/carbon": "^2.29"
+        "drupal/core": "^8.9 || ^9",
+        "php": ">=7.3",
+        "nesbot/carbon": "^2.47"
```

Comment:
Why are you increasing this (and all other deps that don't have to be increased)?